wiki/en/Include-Client-Commands.md

```diff
@@ -2,5 +2,5 @@
 - `--mutemyown` Prevent me from hearing what I play in the server mix (headless only)                                                      
 -  `-c` or `--connect`  Connect to given server address on startup, format `address[:port]`  
 -  `-j` or `--nojackconnect`  Disable auto JACK connections  
--  `--ctrlmidich`  MIDI controller channel to listen on, control number offset and consecutive CC numbers (channels) and Mute Myself CC number. Format: `channel[;f*][;p*][;s*][;m*][;o]` See [Tips & Tricks](Tips-Tricks-More#using-ctrlmidich-for-midi-controllers)
+-  `--ctrlmidich`  MIDI controller channel to listen on, MIDI control number and count of consecutive CC numbers (Jamulus channels), Mute Myself CC number and device selection option. Format: `channel[;fn[*n]][;pn[*n]][;sn[*n]][;mn[*n]][;on][;dDeviceName]` See [Tips & Tricks](Tips-Tricks-More#using---ctrlmidich-for-midi-controllers).
```

Comment:
```suggestion
-  `--ctrlmidich`  MIDI channel to listen on, Jamulus control + MIDI control number and count of consecutive CC numbers (i.e. Jamulus channels), device selection option. Format: `channel[;fn[*n]][;pn[*n]][;sn[*n]][;mn[*n]][;on][;dDeviceName]` See [Tips & Tricks](Tips-Tricks-More#using---ctrlmidich-for-midi-controllers).
```
The English text was a bit inconsistent with the "Format" bit.  I'm not sure this is explanatory enough but it gives the link to the docs anyway.